lib/detect.js

```diff
@@ -1,31 +1,20 @@
-const Octokit = require('@octokit/rest');
-const core = require('@actions/core');
-const { toChunk, sleep } = require('./utils');
-const { throttling } = require("@octokit/plugin-throttling");
+const core = require("@actions/core");
+const { toChunk, sleep } = require("./utils");
+const octokitWithThrottling = require("./octokitWithThrottling");
 
-const DELAY_BETWEEN_CALLS = 2000;
-const THROTTLING_RETRY_LIMIT = 3;
+const DELAY_BETWEEN_CALLS = 3000;
 
-module.exports = async function detect({ octokit, token, owner, repo, base, head }) {
+module.exports = async function detect({
+  octokit,
+  token,
+  owner,
+  repo,
+  base,
+  head,
+}) {
   if (!octokit) {
```

Comment:
Here was the problem. You added the plugin and it was right! But this instance of octokit was never used because of this "if" statement. Why? well `git-pr-release.js` is the one calling this file:
```javascript
const pulls = await detect({ octokit, token, owner, repo, base, head });
```

As you see it passes octokit so the condition `!octokit` is never reached so the plugin is never used. @markelarizaga

Reply:
OMG, good catch.